draft-ietf-quic-transport.md

```diff
@@ -1296,6 +1291,22 @@ connection ID, it should expect packets on the new connection ID to start at 18.
 A packet with the new connection ID and a packet number of 17 is discarded as
 being in error.
 
+#### Packet Number Gap
+
+In order to avoid linkage, the packet number gap MUST be externally
+indistinguishable from random. The packet number gap for a connection
+ID with sequence number is computed by encoding the sequence number
+as a 32-bit integer in big-endian format, and then computing:
+
+~~~
+Gap = HKDF-Expand-Label(packet_number_secret,
+                        "QUIC packet sequence gap", sequence, 4)
```

Comment:
@ekr's design is best practice here.  I would not be that happy about a design that was less robust than this.  HKDF has a bunch of features that we might not need, but the alternatives would require considerably more analysis before we could be sure that they are OK.